docs/mobile/authoring.md

```diff
@@ -0,0 +1,128 @@
+# Authoring
+
+Writing a DataCamp for Mobile course is a unique experience. The upside of the
+mobile platform is that you can reach a much broader audience because the
+convenience is higher and the barrier to entry is lower. The challenge is that
+there are many more constraints to work within. Primary among these are the
+restricted screen real estate and the lack of an interactive interpreter. These
+constraints require you to think carefully about the way you introduce concepts
+and the examples that you use.
+
+## Getting started
+
+Create and checkout a feature branch to write your course in. For example:
+
+```sh
+git checkout -b chapter-1
+```
+
+> #### info::Note
+> You should author content on a feature branch, *not* on the `master` or
+> `development` branches!
+
+You are free to use whatever editing tools you like on your local repository, eg
+Sublime, Vim, Emacs, etc. It can be helpful to use an IDE that allows you to
+expand snippets, and to write a snippet expansion for different exercise
+types. Bonus points if your IDE of choice allows you to automatically generate
+UUIDs for your exercises.
+
+Visit the [exercises documentation](exercises/readme.md) to see how exercises
+are written and see some examples.
+
+The best workflow is generally to open an appropriate interpreter side by side
+with the content file, so that you can play with code and quickly insert it into
+your exercises. This might be IPython, R, or a SQL client like `psql` or
+`sqlite`.
+
+Currently, all UUIDs (eg for chapters, lessons, and exercises) must be generated
+manually. This annoyance can be alleviated by finding a UUID generator package
+for your text editor or IDE of choice. Some examples:
+
+* [Emac](https://github.com/kanru/uuidgen-el)
+* [Vim](https://github.com/kburdett/vim-nuuid)
+* [Sublime Text](https://github.com/SublimeText/GenerateUUID)
+
+## Strategy
+
+It's a good idea to start a lesson by creating a scaffold. First write a one
+line comment for each topic that you want to address in the lesson. For example:
+
+```yaml
+# introduce datasets
+
+# SELECT 1 column
+
+# SELECT multiple columns
+
+# SELECT *
+
+# LIMIT
+
+# SELECT DISTINCT
+```
+
+Then write an exercise for each topic. You can familiarize yourself with how
+exercises work by perusing the [exercises
+documentation](exercises/readme.md). Don't be afraid to split one topic into 2
+or even 3+ exercises if it's difficult to teach something in a single
+exercise. One great strategy is to hit the same topic from multiple angles by
+using different exercise types back to back. You will find that certain subjects
+naturally lend themselves to certain exercise types, but in general you should
+try to vary the exercise types in a lesson.
+
+## Working with YAML
```

Comment:
Provide links to other YAML resources. I suggest linking to an introductory tutorial, a syntax cheatsheet, and a parser.

There are loads of tutorials; this one is nice because it is short and also links to other resources.

https://gettaurus.org/docs/YAMLTutorial/#YAML-Gotchas

I like this cheatsheet.

https://kapeli.com/cheat_sheets/YAML.docset/Contents/Resources/Documents/index

This parser is ace for debugging.

https://yaml-online-parser.appspot.com

Reply:
Oh, reading on I see you already linked to the parser.